drivers/udevSR.py

```diff
@@ -148,6 +148,9 @@ def load(self, location):
             iso8601 = time.strftime("%Y-%m-%dT%H:%M:%SZ", time.gmtime(s[stat.ST_CTIME]))
             self.sm_config['hotplugged_at'] = iso8601
 
+            if os.path.islink(self.path):
```

Comment:
I would say that the `if os.path.islink(self.path):` is unnecessary here and it should just always do `self.path = os.pth.realpath(self.path)`

Reply:
Agreed and fixed - I guess I wanted to make sure the behavior of `os.path.realpath` wouldn't negatively affect the function for non-symlink situations, but it should just return the same thing if it's a true path.